### PR TITLE
Changes for deployment to prod-like environments

### PR DESCRIPTION
**Pull request by little9**
This adds Capistrano, switches the database to MySQL, changes the `SECRET_KEY_BASE`, and
has other changes that needed to be made to deploy the application via Ansible. 

`SECRET_KEY_BASE` will be stored in LastPass.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,14 +5,16 @@ ruby '2.6.5'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.0'
-# Use sqlite3 as the database for Active Record
-gem 'sqlite3', '~> 1.4'
```
Comment (ewlarson) [inline Gemfile:9]: Per our Slack chat yesterday, let's keep sqlite3 for dev and test environments. We'll use mysql in production only.

**Review by ewlarson — CHANGES_REQUESTED** [review]
Great work here! Just the one change we discussed yesterday.

File: config/database.yml [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
 # Do not set this db to the same as development or production.
 test:
-  <<: *default
+  adapter: sqlite3
+  pool: 1
+  timeout: 5000
   database: db/test.sqlite3
 
-production:
+gproduction:
```
Comment (ewlarson) [inline config/database.yml:27]: An errant 'g' here?

**Review by ewlarson — CHANGES_REQUESTED** [review]
Think just that 'g' needs fixing here.

File: config/database.yml (hunk 2 above)
Comment (little9) [reply config/database.yml]: Thank you for catching this!

**Pull request merged**